modules/installation-configuring-shiftstack-single-ipv6.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// TBD
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-shiftstack-single-ipv6_{context}"]
+= Configuring a cluster with single-stack IPv6 networking
+
+You can create a single-stack IPv6 cluster on {rh-openstack-first}.
+
+.Prerequisites
+
+* Your {rh-openstack} deployment has an existing IPv6 subnet.
+* DNS is configured for the existing IPv6 subnet.
+* The IPv6 subnet is added to a {rh-openstack} router, and the router is configured to send router advertisements (RAs). 
```

Comment:
For clarity, I would include https://github.com/openshift/installer/pull/9323 on this point.

Reply:
oh this is the same that Maysa commented on https://github.com/openshift/openshift-docs/pull/85554/files#r1901178233. Max, I let you decide where to put it :)